common/static/js/vendor/image-zoom-tool.js

```diff
@@ -0,0 +1,324 @@
+/*
+ * edX Image Zoom Tool
```

Comment:
I was thinking it was a vendor file, supplied by us. The way it's "downloaded" for use in Studio suggests that it's not something that's just part of our platform - course authors have to download this file and then upload it to their course in order to use this tool. Where would you advise it go, if not in vendors?